### PR TITLE
Update error messages

### PR DESCRIPTION
**Pull request by HonzaCuhel**

### CODE REVIEW

**Review by tersekmatija — CHANGES_REQUESTED** [review]

File: client/src/store.js [hunk 1]
```diff
@@ -66,12 +66,12 @@ export const upload = createAsyncThunk(
...
+          throw Error("Error while loading model (This may be caused by trying to convert a newer version of YoloV6 - release 2.0 or 3.0, if that is the case, try to convert using the `YoloV6 (R2, R3)` option, or by trying to convert the latest release 4.0 that isn't supported yet, we are working on that).");
         case 518:
           let errorData = JSON.parse(String.fromCharCode.apply(String, new Uint8Array(error.response.data)))
           throw Error(errorData['message']);
         case 519:
-          throw Error("Error while loading model (This may be caused by trying to convert an older version of YoloV6 - release 1.0, if that is the case, try to convert using the `YoloV6 (legacy - R1)` option).");
+          throw Error("Error while loading model (This may be caused by trying to convert an older version of YoloV6 - release 1.0, if that is the case, try to convert using the `YoloV6 (R1)` option, or by trying to convert the latest release 4.0 that isn't supported yet, we are working on that).");
```
[the remark this answers is not in the document]
Comment (HonzaCuhel) [reply client/src/store.js:74]: Agree, will change it.

**Comment by HonzaCuhel** [comment]
Updated.

File: client/src/store.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (HonzaCuhel) [reply client/src/store.js]: Updated

**Review by tersekmatija — APPROVED** [review]

**Comment by HonzaCuhel** [comment]
So what about this: `Error while loading model (This may be caused by trying to convert either the latest release 4.0 that isn't supported yet, or by releases 2.0 or 3.0, in which case, try to convert using the YoloV6 (R2, R3) option)`? Is this better? @tersekmatija

**Comment by tersekmatija** [comment]
Yeah, let's go with that. @HonzaCuhel

**Pull request merged**